--- efax-gtk-3.0.10.org/efax-gtk.desktop	2006-06-22 10:17:01.000000000 +0200
+++ efax-gtk-3.0.10/efax-gtk.desktop	2006-06-22 10:21:46.000000000 +0200
@@ -2,10 +2,13 @@
 Name=Efax-gtk
 Name[zh_TW]=傳真
 Comment=GUI front end for the efax fax program
+Comment[pl]=Graficzny interfejs programu efax
 Comment[zh_TW]=efax 傳真軟體的 GUI 前端程式
 TryExec=efax-gtk
 Exec=efax-gtk
 Icon=stock_send-fax.png
 Terminal=false
 Type=Application
-Categories=Application;Office;GTK
+Categories=GTK;Office;X-Misc;
+
+# vim: encoding=utf-8
